cmd/minikube/cmd/start.go

```diff
@@ -1585,17 +1591,24 @@ func createNode(cc config.ClusterConfig, existing *config.ClusterConfig) (config
 	// Create the initial node, which will necessarily be a control plane
 	if existing != nil {
 		cp, err := config.PrimaryControlPlane(existing)
-		cp.KubernetesVersion = getKubernetesVersion(&cc)
-		cp.ContainerRuntime = getContainerRuntime(&cc)
 		if err != nil {
 			return cc, config.Node{}, err
 		}
+		newVer, err := getKubernetesVersion(&cc)
+		cp.KubernetesVersion = newVer
```

Comment:
```suggestion
		cp.KubernetesVersion, err = getKubernetesVersion(&cc)
```
There shouldn't be anything between the declaration of an error and the error check, so `cp.KubernetesVersion = newVer` should be after the error check. But because the `newVer` is only being used to set `cp.KubernetesVersion` we can just set it directly